tests/run-pass/concurrency/tls_lib_drop_single_thread.stderr

```diff
@@ -0,0 +1,2 @@
+Continue main.
+Dropping: 5
```

Comment:
This test seems to fail on Windows (says CI).
I verified locally that it passes on current master, so this is a regression that should be fixed. (You can use `./miri run --target i686-pc-windows-gnu` to run tests on Windows no matter your host OS.)